fiftyone/core/odm/utils.py

```
@@ -10,6 +10,7 @@
 import json
 import numbers
 import sys
+from typing import Any, Dict, Iterable, List, Union
```

Ruff (0.14.7)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)